### PR TITLE
Add stale PR section on review process

### LINKED ISSUE #5152: [docs] Add stale section on FAQs
So far we didn't explain how recipes are closed when getting stale, people need to read stale configuration file.

Also we didn't explain what should be done when getting stale, what labels should be used, changes on PR, getting help from PR author ...

### PR DESCRIPTION
**Pull request by uilianries**
Stale is not documented, we only have its configuration file.

closes #5152

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: docs/review_process.md [hunk 1]
```diff
@@ -153,3 +153,7 @@ This also blocks further merges until upload is finished. It also explains why n
 repository.  Instead, it's updated by the conan center CI job as its own step. The metadata from the conan repository is
 converted to the format the web-front-end understands and then sent to it as a scheduled update. As a result, there may occasionally be delays in updating the web-front-end.
 That may explain the fact there are moments when the information showed in the frontend doesn't match the actual state on the ConanCenter repository.
+
+## Stale PRs
+
+Conan Center Index uses [stale bot](https://github.com/probot/stale) to close abandoned pull requests. It's configured by [stale.yml](.github/stale.yml). When a pull request gets stale, [official reviewers](#official-reviewers) are allowed to update authot's PR, fixing or changing what is considered necessary to be accepted on review process.
```
Comment (jgsogo) [inline docs/review_process.md:159]:
```suggestion
Conan Center Index uses [stale bot](https://github.com/probot/stale) to close abandoned pull requests. It's configured by [stale.yml](.github/stale.yml). When a pull request gets stale, [official reviewers](#official-reviewers) are allowed to update author's PR, fixing or changing what is considered necessary to be accepted on review process.
```

As reviewers won't be able to push to author's branch (only those with write access to the repository can do it), I would use something more generic: 

_"When a pull request gets stale, we encourage anyone to take ownership of the PR (even submit changes to the author's branch if possible) so existing work doesn't get lost when the pull request is closed without merging."_

Comment (SSE4) [inline docs/review_process.md:159]:
```suggestion
Conan Center Index uses [stale bot](https://github.com/probot/stale) to close abandoned pull requests. It's configured by [stale.yml](.github/stale.yml). When a pull request gets stale, [official reviewers](#official-reviewers) are allowed to update author's PR, fixing or changing what is considered necessary to be accepted on review process.
```

Comment (uilianries) [reply docs/review_process.md]: Done, updated to anyone

Comment (uilianries) [reply docs/review_process.md]: Done

**Review by jgsogo — APPROVED** [review]

**Review by czoido — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**